assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/clip/clip_embeddings_mlflow_wrapper.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""MLflow PythonModel wrapper class that loads the MLflow model, preprocess inputs and performs inference."""
+
+import mlflow
+from PIL import Image
+import pandas as pd
+import torch
+import tempfile
+
+import os
+MODEL_DIR = os.path.dirname(__file__)
```

Comment:
If this is for unit tests to pass, can we add this to the unit test function, instead of adding it here.